### PR TITLE
(maint) Release Prep 1.1.0

### PR DESCRIPTION
**Pull request by david22swan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+
+### Added
+- Functions added for onverting to and from bytes.
+- Tasks added for common actions that use types and providors.
+- Plan for one touch expanding, expanded.
+- Support has been added for LVM Thin Volumes.
+
+### Changed
+- Facts have been structured.
+- Providor modified to allow `puppet resource` to work with `logical_volumes`.
```
Comment (pmcmaw) [inline CHANGELOG.md]: Provider

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+### Summary
+This is a small release in order to add facts, functions, tasks and plans.
+
+### Added
+- Functions added for onverting to and from bytes.
+- Tasks added for common actions that use types and providors.
+- Plan for one touch expanding, expanded.
+- Support has been added for LVM Thin Volumes.
+
+### Changed
```
Comment (pmcmaw) [inline CHANGELOG.md]: Changed header means 'Backwards Incompatible' therefore these are probably features.

Comment (pmcmaw) [reply CHANGELOG.md]: 'Thin Volumes' I dont think you need the uppercase here.

File: CHANGELOG.md (hunk 1 above)
Comment (david22swan) [reply CHANGELOG.md]: Fixed the spelling

File: CHANGELOG.md (hunk 2 above)
Comment (david22swan) [reply CHANGELOG.md]: Changed to `LVM thin volumes`

Comment (david22swan) [reply CHANGELOG.md]: `Added` and `Changed` merged into `Features`. `Fixed` changed to `Bugfixes`

**Pull request merged**